src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -1656,20 +1655,13 @@ public void testCachedFile() throws Exception {
         checker.destroy();
     }
 
-    @SuppressForbidden
     @Test
     public void testUnmappableCharacters() throws Exception {
         final String[] expected = {
-            "4: " + getCheckMessage(LineLengthCheck.class, MSG_KEY, 75, 238),
+            "4: " + getCheckMessage(LineLengthCheck.class, MSG_KEY, 80, 100),
         };
 
-        final DefaultConfiguration checkConfig = createModuleConfig(LineLengthCheck.class);
-        checkConfig.addProperty("max", "75");
-
-        final DefaultConfiguration checkerConfig = createRootConfig(checkConfig);
-        checkerConfig.addProperty("charset", "IBM1098");
-
-        verify(checkerConfig, getPath("InputCheckerTestCharset.java"), expected);
+        verifyWithInlineXmlConfig(getPath("InputCheckerTestCharset.java"), expected);
```

Comment:
you have to put the config comment in xml format in the beginning of this input file

Reply:
Why are we using verifyWithInlineXmlConfig instead of verifyWithInlineConfig?